src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -453,16 +481,16 @@ private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
             {
                 Output.Error(false, CommandLineResources.TestRunAborted);
             }
-            else if (this.testOutcome == TestOutcome.Failed && this.testsTotal > 0)
+            else if (testsFailed > 0)
```

Comment:
>testsFailed [](start = 21, length = 11)

We should revert this one.
We could have a passing tests and then we some failure occurred.
TestOutcome is set to failed in case some error occurs during the run, it is not necessary that we would have failing test.

Reply:
Ok. Will fix this. Didnt see any  UT failing though.. will add one.